dxcherry/src/tree/payload.ts

```diff
@@ -0,0 +1,37 @@
+import TreeDataProvider from './dataProvider';
+import Git from '../git/client';
+import { logInfo } from '../info';
+
+interface TreePayload {
+  title: string,
+  description: string,
+  versions: string[],
+  reviewers: string[],
+  labels: string[]
+};
+
+async function getTreePayload (treeDataProvider: TreeDataProvider): Promise<TreePayload | undefined> {
+  logInfo('Payload is getting');
+
+  const lastCommit = await Git.getLastCommit();
+  logInfo(`Last commit: ${JSON.stringify(lastCommit)}`);
+
+  if (!lastCommit) {
+    // TODO: when is there no last commit?
```

Comment:
When branch is created in the new repository without commits at all.

Reply:
ok thx